src/database/flashcardManager.js

```diff
@@ -0,0 +1,39 @@
+import db from "./db";
```

Comment:
Ten kod wrzuc do flashcardsManager, tam jest jedna metoda to ja mozesz zostawic, bo nie wiem czy jest potrzebna. Ale robiac taki service to duplikujemy rzeczy ^ pisalem o tym wyzej juz